mne/preprocessing/ica.py

```diff
@@ -575,13 +574,12 @@ def _transform_evoked(self, evoked):
         if not hasattr(self, 'mixing_matrix_'):
             raise RuntimeError('No fit available. Please first fit ICA')
 
-        picks = pick_types(evoked.info, include=self.ch_names, exclude=[],
-                           ref_meg=False)
+        picks = [evoked.ch_names.index(k) for k in self.ch_names]
```

Comment:
It makes sure to only use the channels used in ICA, right?
Any other consequences?